### PR TITLE
feat(test): testing implementation

### PR DESCRIPTION
**Pull request by cdaringe**
# problem statement

we have no tests!

# solution

add tests!

# discussion

tbd

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +32,10 @@
     "glob": "^7.1.1",
     "gulp": "^3.9.1",
     "html-loader": "^0.4.4",
+    "httpster": "^1.0.3",
     "javascript-natural-sort": "^0.7.1",
     "jest": "^19.0.2",
+    "jquery": "^3.2.1",
```
Comment (cdaringe) [inline package.json]: unused, remove

File: package.json [hunk 2]
```diff
@@ -53,6 +55,7 @@
     "standard": "^9.0.0",
     "style-loader": "^0.13.2",
     "url": "^0.11.0",
+    "webjerky": "^0.2.1",
```
Comment (cdaringe) [inline package.json]: mmmmmmm

**Comment by cdaringe** [comment]
remove `run/` from commits, & rebase

**Comment by cdaringe** [comment]
arg, i need to move on to another project and come back to this.  i can't get it to work in CI?

@jhegg, if you have time sometime soon, will you pull this branch & `yarn test`? it works on my machine.  you need sauce creds.  i can get those to you

**Comment by cdaringe** [comment]
i got desperate and SO'd http://stackoverflow.com/questions/43284281/sauceconnect-on-codeship

**Comment by eddier** [comment]
This looks good.  @cdaringe I was able to run all of this locally like you were - I can take a look at the CI issue if you can send over the creds?

**Comment by cdaringe** [comment]
emailed!

**Pull request closed without merging**